src/pages/home/report/ReportDropUI.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {createPortal} from 'react-dom';
+import {View} from 'react-native';
+import CONST from '../../../CONST';
+import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+import DragAndDropIcon from '../../../../assets/images/drag-and-drop.svg';
+import styles from '../../../styles/styles';
+import Text from '../../../components/Text';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+};
+
+const DropZone = ({children}) => createPortal(
+    children,
+    document.getElementById(CONST.REPORT.DROP_NATIVE_ID),
+);
+
+const ReportDropUI = props => (
+    <DropZone>
+        <View style={[styles.dragAndDropOverlay, styles.alignItemsCenter, styles.justifyContentCenter]}>
+            <View style={[styles.alignItemsCenter, styles.justifyContentCenter]}>
```

Comment:
I'm suspicious that this child view on line 22 shouldn't be necessary. Please check if you can remove it and things work just the same.

Reply:
Right, updated